plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduDistributedQueries.java

```diff
@@ -121,6 +121,30 @@ public void testDelete()
         // TODO Support these test once kudu connector can create tables with default partitions
     }
 
+    @Override
+    public void testDeleteWithComplexPredicate()
+    {
+        // TODO Support these test once kudu connector can create tables with default partitions
```

Comment:
Maybe for Kudu we could use previous assertion?

Reply:
i.e. keep the override but add an assertion to catch this once Kudu starts supporting CREATE with default partitions?

```java
@Override
public void testDeleteWithComplexPredicate()
    if (!hasBehavior(SUPPORTS_DELETE)) {
        assertQueryFails("DELETE FROM nation", "This connector does not support deletes");
        return;
    }
}
```

Did I understand correctly?

Reply:
Yes